docs/configuration/pepr-policies.md

```diff
@@ -9,7 +9,7 @@ weight: 3
 ### Pepr Policy Exemptions {#pepr-policy-exemptions}
 These policies are based on the [Big Bang](https://p1.dso.mil/services/big-bang) policies created with Kyverno. You can find the source policies [here](https://repo1.dso.mil/big-bang/product/packages/kyverno-policies), Policy Names below also have links to the referenced Big Bang policy.
 
-Exemptions can be specified by a [UDS Exemption CR](../operator/README.md). These take the place of Kyverno Exceptions.
+Exemptions can be specified by a [UDS Exemption CR](../uds-operator.md#exemption). These take the place of Kyverno Exceptions.
```

Comment:
The docs site actually gets mad about this link (it ends up routing to https://uds.defenseunicorns.com/core/configuration/operator/README.md currently). We probably should make this a fully qualified link rather than a local ref, but otherwise this change makes sense :)

Reply:
```suggestion
Exemptions can be specified by a [UDS Exemption CR](https://uds.defenseunicorns.com/core/configuration/uds-operator/#exemption). These take the place of Kyverno Exceptions.
```

Reply:
Resolved a different way.